source/MRViewer/MRUIRectAllocator.cpp

```diff
@@ -0,0 +1,210 @@
+#include "MRUIRectAllocator.h"
+
+#include "MRViewer/MRViewer.h"
+
+#include <imgui_internal.h>
+#include <spdlog/spdlog.h>
+
+namespace MR::UI
+{
+
+RectAllocator::RectAllocator() {}
```

Comment:
= default? 
is it moved here to avoid inline?

Reply:
Mostly out of habit. Should be a bit faster to compile too, if multiple files need to construct this.